includes/pages/user_shifts.php

```diff
@@ -374,7 +374,7 @@ function ical_hint()
 
     return heading(__('iCal export and API') . ' ' . button_help('user/ical'), 2)
         . '<p>' . sprintf(
-            __('Export your own shifts. <a href="%s">iCal format</a> or <a href="%s">JSON format</a> available (please keep secret, otherwise <a href="%s">reset the api key</a>).'),
+            __('Export your own shifts. <a href="%s" target="_blank" rel="noopener">iCal format</a> or <a href="%s" target="_blank" rel="noopener">JSON format</a> available (please keep secret, otherwise <a href="%s" target="_blank" rel="noopener">reset the api key</a>).'),
```

Comment:
In that case the noopener is not needed as they link to the same site ^^

Also the api key reset should not have a target=_blank as its the same page and i don't see a usecase where it would help to open it in another tab?

Reply:
done